en/02_Development_environment.adoc

```diff
@@ -182,7 +192,14 @@ You'll also need `make`.
 
 === Vulkan Packages
 
-The most important components you'll need for developing Vulkan applications on Linux are the Vulkan loader, validation layers, and a couple of command-line utilities to test whether your machine is Vulkan-capable:
+LunarG provides the Vulkan SDK for 64-bit Linux distribution. Installation depends on your distribution and installation:
+
+- link:https://vulkan.lunarg.com/doc/view/latest/linux/getting_started_ubuntu.html[Getting Started with the Ubuntu Vulkan SDK]
+- link:https://vulkan.lunarg.com/doc/view/latest/linux/getting_started.html[Getting Started with the Linux Tarball Vulkan SDK]
```

Comment:
For Linux, I recommend to most programmers to create a directory to hold Vulkan then a soft link so all paths are always the same after upgrades.  i.e.

mkdir -p ~/Vulkan_SDK && pushd ~/Vulkan_SDK
then every time you get vulkan via new udpate even:
tar -xzf <pathtodownload>/vulkan.tar.gz
ln -s latest x.x.x

Now whenever you set the Vulkan Path anywhere in a project, you point it to ~/Vulkan_SDK/latest.  You can now update/downgrade Vulkan and just update the softlink to keep the same path everywhere.